src/model/specific_path/PathPart.java

```diff
@@ -0,0 +1,47 @@
+package model.specific_path;
+
+import java.awt.Point;
+import java.util.ArrayList;
+
+
+
+public class PathPart {
+	
+	private ArrayList<Point> points;
+	private LineType type;
+	
+	public PathPart(Point point, LineType type) {
+		this.points = new ArrayList<Point>();
+		this.points.add(point);
+		this.type = type;
+	}
+
+	public PathPart(ArrayList<Point> points, LineType type) {
+		super();
+		this.points = points;
+		this.type = type;
+	}
+
+
+
+	public ArrayList<Point> getPoints() {
+		return points;
+	}
+
+
+
+	public void setPoints(ArrayList<Point> points) {
```

Comment:
One suggestion, we would not expose `points` with this setter. And add a method `addPoint(Point p)` because I found that the line `bez.getPoints().add(new Point(400, 350))` in `Draw` is a little bit confusing.

**Any way, all works well. I am ok to merge what you got now.**